descriptive-link-5effbb.md

```diff
@@ -0,0 +1,179 @@
+---
+id: 5effbb
+name: Link text is descriptive
```

Comment:
Would it make sense to call it "Link in context is descriptive" or something like that to make it clear that it is for 2.4.4 and not 2.4.9. 
Basically, the 2.4.9 rule will be REALLY easy to write whenever this rule is done. It's just removing the context part. So I think we should already now allow room for creating this other rule.